src/parsers/CMakeLists.txt

```diff
@@ -39,3 +39,16 @@ install(
     RUNTIME DESTINATION "bin"
 )
 install(FILES ${HEADERS} DESTINATION ${INCLUDE_INSTALL_DESTINATION}/parsers)
+
+
+add_executable(urdf_yaml_converter urdf_yaml_conv.cpp)
+target_link_libraries(urdf_yaml_converter RBDyn::Parsers)
+set_target_properties(urdf_yaml_converter PROPERTIES COMPILE_FLAGS "-DRBDYN_PARSERS_EXPORTS")
```

Comment:
Why? It's a utility to convert between formats, it needs the parsers library for that.

Reply:
Adding `-DRBDYN_PARSERS_EXPORTS` will change the API declaration to `__declspec(dllexport)` but we want `__declspec(dllimport)` since we are consumers of the library

Reply:
I changed it to `-DRBDYN_PARSERS_DLLIMPORT`